### ISSUE TITLE
Add dry run option

### ISSUE THREAD

**Issue by madnight** [sindresorhus/replace-in-files-cli#1]
If the dry run option is set, then no changes should happen, instead it should list all changes that would happen.

**Comment by sindresorhus** [comment]
Can you show an example of how that would look like? How would it list the changes?

**Comment by NightMachinary** [comment]
@sindresorhus I imagine sth like `ripgrep` is best; You can then show the replacement in another color. (I suggest using background colors, too.) This dryrun feature is essential for a calm replacement :D.
![image](https://user-images.githubusercontent.com/36224762/79482794-8ebbb400-8026-11ea-9904-6137d6b104f0.png)

**Comment by sindresorhus** [comment]
@NightMachinary I like it 👍

---

> This dryrun feature is essential for a calm replacement :D.

Sure. However, this CLI is really meant to be used in run scripts of packages/apps, where you're already using Git, so you can always just revert the commit / working directory if you did something wrong.

**Comment by madnight** [comment]
Yup, I also like @NightMachinary proposal. Maybe one could also add a interactive mode, where the user has to confirm (y/n) each replacement.
